terraform/provider/provider.go

```diff
@@ -42,7 +42,7 @@ import (
 
 const (
 	// minServerVersion is the minimal teleport version the plugin supports.
-	minServerVersion = "6.1.0-beta.1"
+	minServerVersion = "15.0.0-0"
```

Comment:
What is the "-0" for? Is it to avoid maching betas/RCs?

Reply:
it is to match the beta/rc. Go's semver will consider 15.0.5-dev below 15.0.0 if you don't explicitly add the `-0` part